src/main/java/org/galatea/starter/domain/AVDay.java

```diff
@@ -0,0 +1,65 @@
+package org.galatea.starter.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+//reference: https://stackoverflow.com/questions/47263236/how-to-parse-json-object-array-with-jackson-into-dto-in-springboot
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AVDay{
+
+  @JsonProperty("1. open")
+  private double open;
+
+  @JsonProperty("2. high")
+  private double high;
+
+  @JsonProperty("3. low")
+  private double low;
+
+  @JsonProperty("4. close")
+  private double close;
+
+  @JsonProperty("5. volume")
+  private double volume;
+
+  public double getOpen() {
+    return open;
+  }
+
+  public void setOpen(double open) {
```

Comment:
Remove getters and setters, replace with Lombok ```@Data``` annotation.
https://projectlombok.org/features/Data